### PR TITLE
`new-badges` added and did some changes 

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**

### CODE REVIEW

**Comment by gurjeetsinghvirdee** [comment]
> _No description provided._

PS: By mistake, I wrote Funday instead of Funding, Sorry

File: README.md [hunk 1]
```diff
@@ -482,6 +486,7 @@ Badge | URL
 ------------ | -------------
 <img src="https://img.shields.io/badge/Adobe%20Dreamweaver-072401?style=for-the-badge&logo=Adobe%20Dreamweaver&logoColor=34F400" />| `https://img.shields.io/badge/Adobe%20Dreamweaver-072401?style=for-the-badge&logo=Adobe%20Dreamweaver&logoColor=34F400`
 <img src="https://img.shields.io/badge/Android_Studio-3DDC84?style=for-the-badge&logo=android-studio&logoColor=white" />| `https://img.shields.io/badge/Android_Studio-3DDC84?style=for-the-badge&logo=android-studio&logoColor=white`
+<img src="https://img.shields.io/badge/apache%20netbeans%20IDE-1B6AC6?style=for-the-badge&logo=apache%20netbeans%20IDE&logoColor=white" /> | `https://img.shields.io/badge/apache%20netbeans%20IDE-1B6AC6?style=for-the-badge&logo=apache%20netbeans%20IDE&logoColor=white`
```
Comment (mrhrifat) [inline README.md:489]: It will be `Apache Netbeans` See [here](https://netbeans.apache.org)

**Review by mrhrifat — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mrhrifat) [inline README.md:489]: It will be `Apache Netbeans`. See [here](https://netbeans.apache.org)

**Review by mrhrifat — CHANGES_REQUESTED** [review]
I mistakenly click on `Approve` instead of `Change`. Please make changes. If I'm wrong feel free to notify me.

File: README.md (hunk 1 above)
Comment (gurjeetsinghvirdee) [reply README.md:489]: > It will be `Apache Netbeans`. See [here](https://netbeans.apache.org)

Thanks for your feedback I will change it

**Review by mrhrifat — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**